docs/understanding-airbyte/connections/README.md

```diff
@@ -1,19 +1,39 @@
-  * [Full Refresh Overwrite](full-refresh-overwrite.md): Sync the whole stream and replace data in destination by overwriting it.
-  * [Full Refresh Append](full-refresh-append.md): Sync the whole stream and append data in destination.
-  * [Incremental Append](incremental-append.md): Sync new records from stream and append data in destination.
-  * [Incremental Deduped History](incremental-deduped-history.md): Sync new records from stream and append data in destination, also provides a deduplicated view mirroring the state of the stream in the source.
+* Sync mode: how streams should be replicated (read and write):
 * Sync schedule: when to trigger a sync of the data
+* Optional transformations: how to convert Airbyte protocol messages (raw JSON blob) data into some other data representations. 
 
-Sync schedules are explained below. For information about catalog selections, see [AirbyteCatalog & ConfiguredAirbyteCatalog](../catalog.md).
+## Sync modes
+
+A sync mode governs how Airbyte reads from a source and writes to a destination. Airbyte provides different sync modes to account for various use cases. To minimize confusion, a mode's behavior is reflected in its name. The easiest way to understand Airbyte's sync modes is to understand how the modes are named.
+
+1.  The first part of the name denotes how the source connector reads data from the source:
+  
+  * Incremental: Read records added to the source since the last sync job.
+      * Method 1: Using a cursor. Generally supported by all sources.
+      * Method 2: Using change data capture. Only supported by some sources. See [CDC](../cdc.md) for more info.
+  * Full Refresh: Read everything in the source.
+
+2. The second part of the sync mode name denotes how the destination connector writes data. This is not affected by how the source connector produced the data:
+
+  * Overwrite: Overwrite by first deleting existing data in the destination.
+  * Append: Write by adding data to existing tables in the destination.
+  * Deduped History: Write by first adding data to existing tables in the destination to keep a history of changes. The final table is produced by de-duplicating the intermediate ones using a primary key.
+
+A sync mode is therefore, a combination of a source and destination mode together. The UI exposes the following options, whenever both source and destination connectors are capable to support it for the corresponding stream:
+* [Full Refresh Overwrite](full-refresh-overwrite.md): Sync the whole stream and replace data in destination by overwriting it.
+* [Full Refresh Append](full-refresh-append.md): Sync the whole stream and append data in destination.
+* [Incremental Append](incremental-append.md): Sync new records from stream and append data in destination.
+* [Incremental Deduped History](incremental-deduped-history.md): Sync new records from stream and append data in destination, also provides a de-duplicated view mirroring the state of the stream in the source.
```

Comment:
@ChristopheDuong can add a note saying that for the first run **Incremental** mode works as a full refresh ?